.editorconfig

```diff
@@ -259,6 +259,6 @@ dotnet_diagnostic.CA2016.severity = suggestion
 
 # Defaults for content in the shared src/ and shared runtime dir
 
-[{**/Shared/runtime/**.{cs,vb},**/microsoft.extensions.hostfactoryresolver.sources/**.{cs,vb}}]
+[{**/Shared/runtime/**.{cs,vb},src/Shared/test/Shared.Tests/runtime/**.{cs,vb},**/microsoft.extensions.hostfactoryresolver.sources/**.{cs,vb}}]
```

Comment:
FYI @Tratcher \ @JamesNK

Reply:
I'm not super sure why our globbing does `**`at the start. Globs are supposed to be relative to the editorconfig file. We'll see if this continues to fail.